cpp/data/eigerfan/src/EigerFan.cpp

```diff
@@ -71,7 +75,9 @@ std::string PadInt(int value) {
 EigerFan::EigerFan()
 : ctx_(EigerFanDefaults::DEFAULT_NUM_THREADS),
   controlSocket(ctx_, ZMQ_ROUTER),
-  forwardSocket(ctx_, ZMQ_PUSH) {
+  forwardSocket(ctx_, ZMQ_PUSH),
+  broker(BROKER_INPROC_ENDPOINT, 1)
```

Comment:
We should probably set this to 8 too. Though I am not sure what this constructor is for.